dtfs-central-api/src/services/state-machines/fee-record/event-handlers/helpers/get-fixed-fee-for-facility.test.ts

```diff
@@ -55,7 +55,7 @@ describe('getFixedFeeForFacility', () => {
           ukefFacilityId: facilityId,
           interestPercentage: 5,
           dayCountBasis: 365,
-          coverStartDate: TODAY.getTime().toString(),
```

Comment:
Is this fine to change / genuinely a `Date`?

Reply:
I had a chat with Josh & he was happy with this - the problem comes from gef, BSS/EWCS and anything in TFM having slightly different dates. Whilst this was happily rendered in TFM, it was breaking gef-ui

Reply:
Hmmm Thinking about it some more, this might be an ISO date

Reply:
These are inserted into the database using the `Facilities` class (portal-api\src\v1\gef\models\facilities.js) which inserts it as a `Date`